packages/ui/v2/core/Shell.tsx

```diff
@@ -136,7 +138,15 @@ const Layout = (props: LayoutProps) => {
       </div>
 
       <div className="flex h-screen overflow-hidden" data-testid="dashboard-shell">
-        {props.SidebarContainer || <SideBarContainer />}
+        {props.settingsSection ? (
+          <div className="hidden lg:block">
+            <SettingsSidebarContainer />
+          </div>
+        ) : props.SidebarContainer ? (
+          props.SidebarContainer
```

Comment:
Since this prop existed before working on the settings shell, I decided it in.

Reply:
Wasn't really what I was trying to explain. You're adding a boolean prop to enclose the container in a div when you can just pass the `SideBarContainer` already enclosed in such div.